analysis/R/association.R

```diff
@@ -147,11 +163,15 @@ UpdatePij <- function(pij, cond_prob) {
   # Returns:
   #   Updated pijs from em algorithm (maximization)
 
+  # NOTE: Not using mclapply here because we have a faster C++ implementation.
```

Comment:
This is already being called inside an mclapply. So the better reason why it should not be is because it would be redundant.